webb_ai/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "ca4c3360-0e0f-4257-a392-8d6e461a3806",
+  "app_id": "webb-ai",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "uninstallation": "README.md#Uninstallation",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "The first AI-enabled reliability engineer",
+    "title": "Webb.ai",
+    "media": [],
+    "classifier_tags": [
+      "Category::AI/ML",
+      "Category::Kubernetes",
+      "Offering::Integration",
```

Comment:
Please update this to reflect the submitted data type as seen [here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags).